action.yml

```diff
@@ -24,5 +28,30 @@ inputs:
     default: 'gpt-4-turbo-preview'
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile' 
\ No newline at end of file
+  using: 'composite'
+  steps:
```

Comment:
🔄 The 'using' field has been updated to 'composite', which allows for more complex workflows. Verify that this change is necessary and beneficial for the action's functionality.